content/_layouts/documentation.html.haml

```diff
@@ -38,7 +38,7 @@ section: doc
           Guided Tour
         %ul
           %li
-            = active_href('doc', 'Index')
+            = active_href('doc', 'Getting Started')
```

Comment:
The title on this page is "Jenkins Documentation".  The first heading under that is "Guided Tour".  

I agree this should not be "Index", but it should match the target page title.

Reply:
I agree that the TOC entry title and page name should match, or closely match, although the issue here (as you've alluded to) is that this page serves multiple purposes:

1. This is the initial landing page of the Jenkins user documentation, which potentially new Jenkins users would see when they first arrive at this documentation. Therefore, this page should cover introductory material about Jenkins, such as:
   - a description of what Jenkins is (which it currently does briefly at the top) and
   - an overview of content in (and with links to) the remainder of the Jenkins user documentation (which currently doesn't exist).
2. It is also the start of the Guided Tour, which really should begin on a new page.

I'll create a new JIRA ticket to address these tasks:
- Creation of a new page with introductory material - perhaps called 'JIRA User Documentation' - that sits above the 'Guided Tour' TOC heading. See [this discussion](https://groups.google.com/forum/#!topic/jenkinsci-docs/fI_gfQ0_ZWk) for some reasoning behind naming this page 'JIRA User Documentation'.
- Reworking the existing page to be content related to getting started with the Guided Tour. Perhaps we could then call this page 'Getting Started with the Guided Tour'?

Anyway, for time being, maybe we can keep the title of this page as 'Getting Started' and I'll replace the heading of 'Guided Tour' (further down the page) with 'Getting Started' (or 'Getting Started with the Guided Tour').

This would also mean removing the page heading 'Getting Started' from the [Create your first Pipeline](https://jenkins.io/doc/pipeline/tour/hello-world/) page :-)

Reply:
Just to add, I like the fact that as soon as I arrive at https://jenkins.io/doc/, the top of the page says 'Jenkins Documentation'.

This tells me immediately what the contents of this site (or sub-site) is.

Based on [the discussion I mentioned above](https://groups.google.com/forum/#!topic/jenkinsci-docs/fI_gfQ0_ZWk) though, it might be better changing this to 'Jenkins User Documentation'.